src/components/MainNav/ProfileDropdown/ProfileDropdown.js

```diff
@@ -89,8 +89,15 @@ class ProfileDropdown extends Component {
 
   createLink(link, index, module) {
     const { stripes } = this.props;
-    const { check } = link;
-    const checkfn = !check ? undefined : (module.getModule()[check] || validations[check]);
+    const { check, route } = link;
+    const checkingFoo = module.getModule()[check] || validations[check];
```

Comment:
Please change variable name. I can't understand what is it and why do we need it?